pkg/setup/setup.go

```diff
@@ -292,7 +292,11 @@ func (s *Setup) installDefaultPlugins(httpClient *httpclient.Client) error {
 	if errCore != nil {
 		return errCore
 	}
-	return errEnterprise
+
+	if errEnterprise != nil {
+		s.logger.Errorf("Cluster setup was successful but dcos-enterprise-cli could not be installed: %s.", errEnterprise)
```

Comment:
Can we print `In order to install the "dcos-enterprise-cli" plugin, make sure your user has the "dcos:adminrouter:package" permission and run "dcos package install dcos-enterprise-cli".`?

Then once we have instructions for air-gapped we'll also add a note about that.